sdk/webpubsub/Azure.Messaging.WebPubSub/src/WebPubSubServiceClient_extensions.cs

```diff
@@ -0,0 +1,367 @@
+        public virtual Response SendToConnection(string connectionId, string message, CancellationToken cancellationToken = default)
+        {
+            RequestOptions options = default;
+            if (cancellationToken != default)
+                options = new RequestOptions() { CancellationToken = cancellationToken };
+
+            return SendToConnection(connectionId, JsonContent, RequestContent.Create(message), options);
+        }
+
+        /// <summary>
+        /// Send message to a group of connections.
+        /// </summary>
+        /// <param name="group">Target group name, which length should be greater than 0 and less than 1025.</param>
+        /// <param name="message"></param>
+        /// <param name="excluded">Excluded connection Ids</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>A <see cref="Response"/> if successful.</returns>
+        public virtual async Task<Response> SendToGroupAsync(string group, string message, IEnumerable<string> excluded = null, CancellationToken cancellationToken = default)
+        {
+            RequestOptions options = default;
+            if (cancellationToken != default)
+                options = new RequestOptions() { CancellationToken = cancellationToken };
+
+            return await SendToGroupAsync(group, JsonContent, RequestContent.Create(message), excluded, options).ConfigureAwait(false);
+        }
+
+        /// <summary>
+        /// Send message to a group of connections.
+        /// </summary>
+        /// <param name="group">Target group name, which length should be greater than 0 and less than 1025.</param>
+        /// <param name="message"></param>
+        /// <param name="excluded">Excluded connection Ids</param>
+        /// <param name="cancellationToken"></param>
+        /// <returns>A <see cref="Response"/> if successful.</returns>
+        public virtual Response SendToGroup(string group, string message, IEnumerable<string> excluded = null, CancellationToken cancellationToken = default)
+        {
+            RequestOptions options = default;
+            if (cancellationToken != default)
+                options = new RequestOptions() { CancellationToken = cancellationToken };
+
+            return SendToGroup(group, JsonContent, RequestContent.Create(message), excluded, options);
+        }
+
+        /// <summary> Check if there are any client connections inside the given group. </summary>
+        /// <param name="group"> Target group name, which length should be greater than 0 and less than 1025. </param>
+        /// <param name="cancellationToken"> The cancellation token to use. </param>
+        public virtual async Task<Response<bool>> GroupExistsAsync(string group, CancellationToken cancellationToken = default)
+        {
+            var options = new RequestOptions() { StatusOption = ResponseStatusOption.NoThrow, CancellationToken = cancellationToken };
```

Comment:
Have we GA'ed `StatusOption` yet?  I don't love that name.

Reply:
We haven't GA'd it yet - it and `RequestOptions` are still both in `Azure.Core.Experimental`